### PR TITLE
BAP-113 Bindings refactoring

### PR DESCRIPTION
**Pull request by SergeyRuzin**

### CODE REVIEW

File: src/Behavioral.Automation/Bindings/ListBinding.cs [hunk 1]
```diff
@@ -57,8 +55,8 @@ public void CheckListHaveItems(IListWrapper list, string behavior, Table table)
         /// | Test value 1 |
         /// | Test value 2 |
         /// </example>
-        [Given("(.+?) (contains|does not contain) the following items:")]
-        [Then("(.+?) should (contain|not contain) the following items:")]
+        [Given("the (.+?) (contains|does not contain) the following items:")]
```
Comment (vladimir-s-quantori) [inline src/Behavioral.Automation/Bindings/ListBinding.cs:58]: Not sure that these are required changes

File: src/Behavioral.Automation/Bindings/ListBinding.cs [hunk 2]
```diff
@@ -87,7 +85,8 @@ public void CheckListContainsItems(IListWrapper list, string behavior, Table tab
         /// | Test value 1 |
         /// | Test value 2 |
         /// </example>
-        [Then("(.+?) should have in exact order the following items:")]
+        [Given("the (.+?) has in exact order the following items:")]
```
Comment (vladimir-s-quantori) [inline src/Behavioral.Automation/Bindings/ListBinding.cs:88]: Same as previous comment

File: src/Behavioral.Automation/Bindings/TableBinding.cs [hunk 3]
```diff
@@ -149,7 +150,8 @@ public void CheckTableHaveValueInColumn(ITableWrapper table, string behavior, st
         /// <param name="condition">Lesser or greater parameter</param>
         /// <param name="value">Expected value</param>
         /// <example> Then "Test" column values should be greater than "2"</example>
-        [Then("(.*?) values should be (lesser|greater) than \"(.*)\"")]
+        [Given("the (.*?) values are (lesser|greater) than \"(.*)\"")]
```
Comment (vladimir-s-quantori) [inline src/Behavioral.Automation/Bindings/TableBinding.cs:153]: Same

File: src/Behavioral.Automation/Bindings/TableBinding.cs [hunk 4]
```diff
@@ -167,7 +169,8 @@ public void CheckTableHaveValueInColumn(ITableWrapper table, string behavior, st
         /// <param name="value1">First boundary</param>
         /// <param name="value2">Second boundary</param>
         /// <example>Then "Test" column values should be between "5" and "10"</example>
-        [Then("(.*?) values should be (between|not between) \"(.*)\" and \"(.*)\"")]
+        [Then("the (.*?) values are (between|not between) \"(.*)\" and \"(.*)\"")]
```
Comment (vladimir-s-quantori) [inline src/Behavioral.Automation/Bindings/TableBinding.cs:172]: Same

**Review by andrey-brovko — APPROVED** [review]

**Review by alspozdnyakov — APPROVED** [review]

File: src/Behavioral.Automation/Bindings/ListBinding.cs (hunk 1 above)
Comment (SergeyRuzin) [reply src/Behavioral.Automation/Bindings/ListBinding.cs:58]: Discussed to leave changes as is

File: src/Behavioral.Automation/Bindings/TableBinding.cs (hunk 3 above)
Comment (SergeyRuzin) [reply src/Behavioral.Automation/Bindings/TableBinding.cs:153]: Discussed to leave changes as is

File: src/Behavioral.Automation/Bindings/TableBinding.cs (hunk 4 above)
Comment (SergeyRuzin) [reply src/Behavioral.Automation/Bindings/TableBinding.cs:172]: Discussed to leave changes as is

**Pull request merged**